test/controllers/billing-accounts.controller.test.js

```diff
@@ -46,6 +47,34 @@ describe('Billing Accounts controller', () => {
     Sinon.restore()
   })
 
+  describe('/billing-accounts/{billingAccountId}', () => {
+    describe('GET', () => {
+      beforeEach(async () => {
+        options = {
+          method: 'GET',
+          url: '/billing-accounts/2e71429d-3fd1-4ed1-a45e-eb5616873018',
+          auth: {
+            strategy: 'session',
+            credentials: { scope: ['billing'] }
+          }
+        }
+      })
+
+      describe('when the request succeeds', () => {
+        beforeEach(async () => {
```

Comment:
This shouldn't need to be `async`
```suggestion
        beforeEach(() => {
```